kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -3,7 +3,7 @@ import urls from 'kolibri.urls';
 // List of channel ids and the app name with the corresponding
 // custom-channel-presentation zip bundle
 export const CustomChannelApps = {
-  '1306b5d7d1ce4b98b0039324f368ce6a': 'pbskids',
+  '1306b5d7d1ce4b98b0039324f368ce6a': 'pbs-kids',
```

Comment:
It's not part of this PR, but maybe we can think about to use the "normalized" channel name instead of the channel-id. I've used that solution for the big thumbnails and I think it's something easier to maintain and update.

Reply:
Hmm. The problem with normalized names is that they are not unique. But yeah it wouldn't matter if 2 channels have the same name.